src/utils/db.ts

```diff
@@ -478,7 +478,8 @@ class DB {
   
   // Creates a channel node and links it the the parent group
   // channelUuid is groupId+channelName
-  async createChannel(channelUuid: String, groupId: String) {
+  async createChannel(channelName: String, groupId: String) {
+    const channelUuid : String = channelName.toString() + "#" + groupId.toString();
```

Comment:
Just here its the groupId first then the channelName. I think you flipped em.

Reply:
Oh whoops, I think the comment is wrong, just fixed it

Reply:
It should be channelname#groupId

Reply:
No the comment is right haha. That how were storing it in dynamo and how storage service expects it to be. So groupId + channelName

Reply:
Oh I see, I was just looking at neo4j, was not considering storage service, ok it is swapped now!